### ISSUE TITLE
deprecated `async_get_registry` warning

### ISSUE THREAD

**Issue by balk77** [B5r1oJ0A9G/teufel_raumfeld#36]
Hi, I am getting the following error. I have no clue how to fix this; would you please be so kind to fix this?
```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:103
First occurred: 19:04:43 (1 occurrences)
Last logged: 19:04:43
Detected integration that uses deprecated `async_get_registry` to access entity registry, use async_get instead. Please report issue to the custom component author for teufel_raumfeld using this method at custom_components/teufel_raumfeld/media_player.py, line 133: entity_registry = await hass.helpers.entity_registry.async_get_registry()
```

**Comment by raimundmuc** [comment]
This change leads to the following error for me (and no media_player entities are created):

`2023-01-08 19:49:28.816 ERROR (MainThread) [homeassistant.components.media_player] Error while setting up teufel_raumfeld platform for media_player`
`Traceback (most recent call last):`
`  File "/home/homeass/homeassistant/lib/python3.10/site-packages/homeassistant/helpers/entity_platform.py", line 289, in _async_setup_platform`
`    await asyncio.shield(task)`
`  File "/home/homeass/.homeassistant/custom_components/teufel_raumfeld/media_player.py", line 133, in async_setup_entry`
`    entity_registry = await hass.helpers.entity_registry.async_get()`
`TypeError: async_get() missing 1 required positional argument: 'hass'`

**Comment by B5r1oJ0A9G** [comment]
Indeed. The corresponding pull request was merged without proper testing. It has been fixed now with  v0.1.12-alpha5.
